java/kafka/hello-world-consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -44,6 +45,12 @@ public static void main(String[] args) {
                 log.info("\tpartition: {}", record.partition());
                 log.info("\toffset: {}", record.offset());
                 log.info("\tvalue: {}", record.value());
+                if (record.headers() != null) {
+                    log.info("\theaders: ");
+                    for (Header header : record.headers().toArray()) {
+                        log.info("\tkey: {}, value: {}", header.key(), new String(header.value()));
```

Comment:
I think double `\t` here will make the output more readable.

Reply:
Agree